dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts

```diff
@@ -26,6 +26,10 @@ export const FormFieldProps = {
     type: Boolean,
     default: false,
   },
+  innerClass: {
+    type: String,
+    default: "short",
+  },
 };
```

Comment:
Let's call this object `BasicFormFieldProps` or `BaseFormFieldProps` instead of just `FormFIeldProps`.
That way it is clear, that those are the "basic" form field props that all form fields will have.

Reply:
Done.

Reply:
Careful: Pls revert this change and redo it again.
You have also renamed the whole file from `FormFieldProps.ts` to `BaseFormFieldProps.ts`
I meant only the first exported object in the class to be renamed to `BaseFormFieldProps`

Reply:
All import path were updated, too. But, no problem. I can revert the file name change.

Reply:
There are some imports in Vue components red now, because the wrong name of the object is used when importing it.  One example:

![image](https://github.com/d-fine/Dataland/assets/99255296/43bd0c6e-d0bb-444f-b844-417e34f3a08f)

Reply:
(Should be `imprt {BaseFormFieldPros} ...`

Reply:
It seem that @MarcinStanczyk7  has fixed this.  Therefore I am closing this.